public/locales/cs/common.json

```diff
@@ -90,6 +90,8 @@
     "why_movapp_license": "Texty, obrázky a audiosoubory Movapp jsou pod <0>licencí CC BY-NC 4.0</0>. (Uveďte původ-Neužívejte komerčně 4.0), což znamená, že je můžete pro vlastní potřeby používat zcela bez omezení. Pokud byste je chtěli dále šířit (v původní nebo v upravené podobě), můžete tak dělat pouze nekomerčním způsobem a s uvedením původních autorů.",
     "why_movapp_origin": "Původní autorkou <0>základní sady slov a frází</0> je Tereza Javornická a Yevheniia Valova (EDUA Group). Tuto sadu udržuje a dále rozšiřuje tým Movapp.",
     "media_mentions_title": "Napsali o nás",
+    "conferences_title": "Konference a přednášky",
+    "conferences_description": "Prezentace <0>Movapp.cz</0> na LUkrainian Conference na Lancaster University (Velká Británie). <br/>- <1>1st LUkrainian Conference (22. 07. 2022)</1><br/>- <2>4th LUkrainian Conference (13. 01. 2023)</2>",
```

Comment:
Remove link from <0>Movapp.cz</0> 
here and in other language variants.

Reply:
it should be fixed now, please take a look